LEAF_Request_Portal/templates/view_reports.tpl

```diff
@@ -688,6 +693,72 @@ function showJSONendpoint() {
 	dialog_message.show();
 }
 
+/**
+ * Purpose: Update Request Titles
+*/
+
+ function changeTitle(form_data) {
+     console.log(form_data);
+	dialog.setContent('Title: <input type="text" id="title" style="width: 300px" name="title" value="<!--{$title|escape:'quotes'}-->" /><input type="hidden" id="CSRFToken" name="CSRFToken" value="<!--{$CSRFToken}-->" />');
+  //ie11 fix
+  setTimeout(function () {
+    dialog.show();
+  }, 0);
+    dialog.setSaveHandler(function() {
+        $.ajax({
+        	type: 'POST',
+        	url: 'api/?a=form/' + form_data.recordID + '/title',
+        	data: {title: $('#title').val(),
+                    CSRFToken: '<!--{$CSRFToken}-->'},
+        	success: function(res) {
+        		if(res != null) {
+                    $('#' + form_data.cellContainerID).fadeOut(400);
+                    $('#' + form_data.cellContainerID).empty().html(res);
+                    $('#' + form_data.cellContainerID).fadeIn(400);
+
+        		}
+                dialog.hide();
+
+        	}
+        });
+    });
+}
+
+/**
+ * Purpose: Create New Row Requests
+ * @param catID - Form ID passing in for new request
+ * @return - Creates new request inline on grid
+ */
+function createRequest(catID) {
```

Comment:
Default value for attribute.

Reply:
per our meeting, holding off on JS default params until transpiler resolution for backwards compatibility